### PR TITLE
Add FAQ resize, edit articles

### PR DESCRIPTION
**Pull request by trispera**
## Proposed changes

Add a FAQ regarding resize of an instance and a volume

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes all tests.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.rahtiapp.fi/origin/) (select your branch from the list).

https://csc-guide-preview.rahtiapp.fi/origin/pouta-resize/support/faq/how-to-resize-in-pouta/

### CODE REVIEW

File: docs/cloud/pouta/vm-flavors-and-billing.md [hunk 1]
```diff
@@ -314,8 +314,9 @@ performance is significantly better.
 
 **io.70GB-700GB:**
 
-NOTE! These servers have non-redundant disks, and you may expect data loss in case of disk failure.
-NOTE! These virtual machines can not be migrated nor resized.
+!!! Note
```
Comment (tolonenj) [inline docs/cloud/pouta/vm-flavors-and-billing.md:317]: If you wish - there are fancy new note-types available: https://docs.csc.fi/ref/#styles-available-with-type-qualifiers

File: docs/cloud/pouta/vm-lifecycle.md [hunk 2]
```diff
@@ -162,7 +162,8 @@ down to other flavors. Scaling down your
 virtual machine when it has less computational workload frees up
 compute resources and saves billing units. Later on,
 depending on your computational workload, you can scale up your virtual
-machines. Please note that you can only resize to the flavors of the
+machines. Please note that you can resize to another flavor from another *family* but it's **highly not recommended!**.
+You may lost data during this process and CSC is not responsible. We recommend to only resize to the flavors of the
```
Comment (tolonenj) [inline docs/cloud/pouta/vm-lifecycle.md:166]: You may lose data...

File: docs/support/faq/how-to-resize-in-pouta.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# How to resize an instance or volume in Pouta
+## Resize an instance
+### Using a snapshot
+We might face performance issue with your initial instance and you'd like to run your instance with a bigger flavor?  
+It's possible to proceed by taking a snapshot of your instance and then boot the new instance with the new flavor using the snapshot.  
+You can find more information about snapshot [here](../../cloud/pouta/snapshots.md) and how to proceed.
+
+### Using the resize functionnality
```
Comment (tolonenj) [inline docs/support/faq/how-to-resize-in-pouta.md:8]: ... functionality

File: docs/support/faq/how-to-resize-in-pouta.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+Wait few minutes and during the process, it will ask you to confirm the resize:
+
+![confirm-resize](img/confirm_resize.png)
+
+After the process finished, you should have your instance with a flavor.
+
+
+## Resize a volume
+!!! Notes  
+    You can only extend a volume. Currently, shrink a volume is not currently supported in Openstack.
```
Comment (tolonenj) [inline docs/support/faq/how-to-resize-in-pouta.md:33]: redundant 'currently' word.

**Review by tolonenj — COMMENTED** [review]
Nice, good work Tristan 🙏 

Not sure do I dare to Approve? Last time I did so, it was merged immediately (efficient guys!) If you wish to check my suggestions first. 🤔

File: docs/cloud/pouta/vm-flavors-and-billing.md (hunk 1 above)
Comment (trispera) [reply docs/cloud/pouta/vm-flavors-and-billing.md:317]: I think a `!!! Note` like I did is enough but thank you for the link 🙂

File: docs/support/faq/how-to-resize-in-pouta.md (hunk 3 above)
Comment (trispera) [reply docs/support/faq/how-to-resize-in-pouta.md]: Fixed

File: docs/cloud/pouta/vm-lifecycle.md (hunk 2 above)
Comment (trispera) [reply docs/cloud/pouta/vm-lifecycle.md]: Fixed

**Review by tolonenj — APPROVED** [review]
All good!

**Pull request merged**